File: eidlogger/src/main/java/no/altinn/eidlogger/dto/EidLogRequest.java
package no.altinn.eidlogger.dto;

import lombok.Getter;
import lombok.Setter;

import java.time.Instant;
import java.util.HashMap;
import java.util.Map;

@Getter
@Setter
public class EidLogRequest {
  private String eventName;
  private Instant eventCreated;
  private String eventDescription;
  private HashMap<String, String> studioData;
}

File: backend/src/Designer/TypedHttpClients/EidLogger/EidLogRequest.cs
using System;
using System.Collections.Generic;

namespace Altinn.Studio.Designer.TypedHttpClients.EidLogger;

public class EidLogRequest
{
    public string EventName { get; set; }
    public DateTime EventCreated { get; set; }
    public string EventDescription { get; set; }
    public Dictionary<string, string> StudioData { get; set; }
}
